service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/StructuredObservationValueMapperTest.java

```diff
@@ -50,6 +54,17 @@ public void When_MappingParsedObservationReferenceRangeJson_Expect_CorrectXmlOut
         assertThat(outputMessage).isEqualToIgnoringWhitespace(expectedOutputMessage);
     }
 
+    @Test
+    public void When_MappingParsedObservationInterpretationJson_Expect_CorrectXmlOutput() throws IOException {
+        String expectedOutputMessage = ResourceTestFileUtils.getFileContent(OUTPUT_XML_WITH_INTERPRETATION);
+
+        var jsonInput = ResourceTestFileUtils.getFileContent(INPUT_JSON_WITH_INTERPRETATION);
+        Observation observation = new FhirParseService().parseResource(jsonInput, Observation.class);
+
+        String outputMessage = XML_OBSERVATION_VALUE_MAPPER.mapInterpretation(observation.getInterpretation());
+        assertThat(outputMessage).isEqualToIgnoringWhitespace(expectedOutputMessage);
```

Comment:
Just based on this https://gpitbjss.atlassian.net/browse/NIAD-1137
We might need to change it to isEqualTo. If you agree, I can help you with that because there might a tricky way to make it work

Reply:
Thanks @antonlebedjko for the call to help with fixing the tests to use `isEqualTo`. 

I have finished replacing the uses of `isEqualToIgnoringWhitespace` with `isEqualTo` in 777f96c and have reformatted the files to include the necessary whitespace as required.

Like with #132, we had to delete the file, paste the contents of the terminal output in to VSCode (not IntelliJ) and save the file again.